### ISSUE TITLE
Server Forcibly Disconnecting Players for Bounding Box

### ISSUE THREAD

**Issue by jordan-moore** [Fuzss/fivefeetsmall#9]
Players Trying to Connect to Server create this output in the server logs and prevents them from joining by forcibly disconnecting them:

```
net.minecraft.util.ReportedException: Ticking player
at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:459) ~[oq.class:?]
at net.minecraft.network.NetHandlerPlayServer.TickCentral_TrueITickableUpdate(NetHandlerPlayServer.java:173) ~[pa.class:?]
at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.TickCentral_TrueITickableUpdate(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
at com.github.terminatornl.tickcentral.api.TickHub.trueUpdate(TickHub.java:48) ~[TickHub.class:3.2]
at com.github.terminatornl.laggoggles.Main.redirectUpdate(Main.java:91) ~[LagGoggles-1.12.2-5.8-132.jar:?]
at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java) ~[NetworkDispatcher$1.class:?]
at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:285) ~[gw.class:?]
at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [oz.class:?]
at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:790) [MinecraftServer.class:?]
at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
Caused by: java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
at net.minecraft.entity.player.EntityPlayer.getBoundingBox(EntityPlayer.java:42340) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayer.isPoseClear(EntityPlayer.java:42269) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayer.updatePose(EntityPlayer.java:42291) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayer.handler$zzj000$updateSize(EntityPlayer.java:42283) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayer.func_184808_cD(EntityPlayer.java) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayer.TickCentral_TrueOnUpdateTick(EntityPlayer.java:288) ~[aed.class:?]
at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:382) ~[oq.class:?]
```

I think this line in particular is relevant:
```
Caused by: java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
```

Removing Aqua Acrobatics prevents such crashes.

[Installed Mod List](https://pastebin.com/D2CLkJQC)

**Comment by jordan-moore** [comment]
Obfuscate and Random patches installed.

config set to 

```
    compat {
        # Only applies when the mod is installed. Disable when there are issues with the mod.
        B:"Mo' Bends Compat"=false

        # Only applies when the mod is installed. Disable when there are issues with the mod.
        B:"Obfuscate Compat"=true

        # Only applies when the mod is installed. Disable when there are issues with the mod.
        B:"Random Patches Compat"=true
    }
```

**Comment by Fuzss** [comment]
That was a simple oversight. A client exclusive method called on the server.
Can you test [this](https://mega.nz/file/KFMknDLL#QAk1WFHrAf3NnzKwkfw2AdwYM4tahgBCFVRNcdacuIw) version if everything works fine now in multiplayer?

**Comment by jordan-moore** [comment]
Looks good on the connection, thanks for the quick reply. I am able to join on the server with the changes.

still some issues though, server log snippet

```
java.io.IOException: An existing connection was forcibly closed by the remote host
at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:1.8.0_261]
at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:1.8.0_261]
at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:1.8.0_261]
at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:1.8.0_261]
at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:1.8.0_261]
at io.netty.buffer.PooledUnsafeDirectByteBuf.setBytes(PooledUnsafeDirectByteBuf.java:288) ~[minecraft_server.1.12.2.jar:?]
at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1100) ~[minecraft_server.1.12.2.jar:?]
at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:372) ~[minecraft_server.1.12.2.jar:?]
at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:123) [minecraft_server.1.12.2.jar:?]
at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:624) [minecraft_server.1.12.2.jar:?]
at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:559) [minecraft_server.1.12.2.jar:?]
at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:476) [minecraft_server.1.12.2.jar:?]
at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438) [minecraft_server.1.12.2.jar:?]
at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [minecraft_server.1.12.2.jar:?]
at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
```

Happened after I took a few steps, client crash.

**Comment by jordan-moore** [comment]
Client Crash Report:
https://pastebin.com/ni22bMSY

**Comment by jordan-moore** [comment]
After a few more relaunches it occurs when a player tries to start walking, crouching seems fine

**Comment by Fuzss** [comment]
Alright, thanks for the feedback. How's [this one](https://mega.nz/file/bAkAzDAA#7k9fhqYJEzQtjdIWAtRhmOjQmOpsZpYcL8-lmfqU_uE) for you? Any more issues?

**Comment by jordan-moore** [comment]
Works great! Thanks for the quick changes. No issues as far as I can tell.

**Comment by Fuzss** [comment]
Thanks for all the feedback. Will release the new version on Curse hopefully soon.

**Comment by Fuzss** [comment]
Released as v1.2 on CurseForge.
